test/testconfig.py

```diff
@@ -116,5 +116,10 @@ def create_transaction(db):
     FEATURE_RH_MARKETPLACE = True
 
     FEATURE_AUTO_PRUNE = True
+    ACTION_LOG_AUDIT_LOGINS = True
```

Comment:
Do you need both `ACTION_LOG_AUDIT_LOGINS` and `ACTION_LOG_AUDIT_LOGIN_FAILURES`?

Reply:
Not necessarily but I was missing this from https://github.com/quay/quay/pull/1866/ and thought it would be good to have it enabled during testing to catch any kind of side effects, even though our current unit tests are explicitly testing for audit events.

Reply:
Not necessarily but I was missing this from https://github.com/quay/quay/pull/1866 and thought it would be good to have it enabled during testing to catch any kind of side effects, even though our current unit tests are explicitly testing for audit events.